api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1235,60 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the number of
+      clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'use_case'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+      created_at:
```

Comment:
I had this in the beginning, but then switched to a single object. There are not so much properties we generate in Nakadi, so the difference will be not so big.
Also if we have two objects, then some day we will have to have two object for EventType as well (as EventType will have creationTime which is generated by nakadi)